### DISCUSSION TITLE
「求助」为什么我发不了图片

### DISCUSSION THREAD

**Discussion by Guievere** [share121/inter-knot#4975] [委托]
还有为啥我这全是英文

**Comment by HLXS-HLXS** [comment]
如果你是想说GitHub全是英文，那我只能回你一句“废话，这是国外网站”
如果你说的是看不见绳网贴子，只有一段英文，那是你网不好，换梯子，电脑推荐steam++，手机自己找

**Comment by HLXS-HLXS** [comment]
至于发不了图片，点paste,drop,or click to add files，选择你要发的图，当图片名结尾是【.jpg…]()】时说明这张图还在上传，等结尾变成网址时才是上传完成，这时在发送
![1709198383897](https://github.com/user-attachments/assets/1d215d8a-d881-428e-9358-3f2be2af2f47)

**Reply by kinich-cute** [reply]
点paste没反应，换头像的edit也是
